pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -28,7 +28,7 @@ import (
 	"k8s.io/klog"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/internal/cache"
-	schedutil "k8s.io/kubernetes/pkg/scheduler/util"
+	channels "k8s.io/kubernetes/pkg/scheduler/internal/channels"
```

Comment:
I think goimports only add alias when the package name is `v1`, it's the first time I saw this and looks weird.

Reply:
+1, that's also my understanding. Please remove this unnecessary alias.